instrumentation/google.golang.org/grpc/interceptor.go

```diff
@@ -269,7 +270,7 @@ func StreamClientInterceptor(tracer trace.Tracer, opts ...Option) grpc.StreamCli
 
 			if err != nil {
 				s, _ := status.FromError(err)
-				span.SetStatus(s.Code(), s.Message())
+				span.SetStatus(codes.Code(s.Code()), s.Message())
```

Comment:
We might want to build out a similar translanation function to [the SDK one](https://github.com/open-telemetry/opentelemetry-go/blob/master/sdk/internal/codes.go) here. The problem with this conversion is that is assumes the underlying integer will remain the same. I don't think it is important to fix in this PR as it will only break when a change to the gRPC codes occurs.

Reply:
https://github.com/open-telemetry/opentelemetry-go-contrib/issues/295

Reply:
I felt slightly dirty doing this, but I'm also pretty sure that `span.SetStatus()` is [not long for this world](https://github.com/open-telemetry/oteps/pull/134), so it may be a moot point.